src/Opaleye/Internal/PGTypesExternal.hs

```diff
@@ -308,3 +322,5 @@ type PGBytea = SqlBytea
 type PGJson = SqlJson
 type PGJsonb = SqlJsonb
 type PGRange = SqlRange
+type PGTSQuery = SqlTSQuery
+type PGTSVector = SqlTSVector
```

Comment:
Existing types named `PG*` are deprecated, so please don't add new ones.

```suggestion
```

Reply:
OK, removed, but how does this compare to the remark above to use `Field PGText`? Should that be `Field SqlText` instead?

Reply:
Oh, yes, I should have said `Field SqlText` above.

Reply:
Is this OK now?

Reply:
Looks fine, thanks.